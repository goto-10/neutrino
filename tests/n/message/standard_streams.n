# Copyright 2014 the Neutrino authors (see AUTHORS).
# Licensed under the Apache License, Version 2.0 (see LICENSE).

import $core;

do {
  $core:stdout.write! ("Foo1\n".view($core:Ascii));
  $core:stderr.write! ("Foo2\n".view($core:Ascii));
  $core:stdout.write! ("Foo3\n".view($core:Ascii));
  $core:stderr.write! ("Foo4\n".view($core:Ascii));
}
