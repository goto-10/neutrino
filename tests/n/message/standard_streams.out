Foo2
Foo4
Foo1
Foo3
